<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Quasar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupConectar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Status da conexão</string>
    </property>
    <widget class="QPushButton" name="buttonConectar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Conectar</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Desconectado</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupConf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuração</string>
    </property>
    <widget class="QGroupBox" name="groupSinal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Sinal</string>
     </property>
     <widget class="QDoubleSpinBox" name="dSpinAmp">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>68</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelAmp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Amplitude</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelOffSet">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Off-set</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dSpinOffSet">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>68</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPeriodo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Período</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dSpinPeriodo">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>68</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboTipoSinal">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelTipoSinal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>59</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tipo:</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonAtualizar">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Atualizar</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupControl">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Controle</string>
     </property>
     <widget class="QRadioButton" name="radioAberta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Malha aberta</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioFechada">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Malha fechada</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupCanalEsc">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Canal de Escrita</string>
     </property>
     <widget class="QComboBox" name="comboCanalEscrita">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="graficoEscrita" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>581</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graficoLeitura" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>991</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupCanalLeitura">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>651</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Canais de Leitura</string>
    </property>
    <widget class="QCheckBox" name="canal_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Canal 0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="canal_1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Canal 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="canal_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Canal 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="canal_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Canal 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="canal_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Canal 4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="canal_5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Canal 5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="canal_6">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Canal 6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="canal_7">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Canal 7</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
